profiles/events.py

```diff
@@ -16,16 +16,23 @@ def send_update_events(publisher: EventPublisher) -> Callable[..., None]:
     def event_handler(sender: Flask, changes: tuple) -> None:  # pylint:disable=unused-argument
         ids = []
 
+        LOGGER.info('Processing event(s)')
+
         for instance, operation in changes:  # pylint:disable=unused-variable
+            LOGGER.info('Event: %s - %s', instance, operation)
```

Comment:
Probably should be something like `'Found operation %s %s', operation, instance`?

Reply:
Changed.